Lib/gftools/fix.py

```diff
@@ -605,28 +605,30 @@ def drop_mac_names(ttfont):
     return changed
 
 
-def _blank_glyph_to_gid1(ttfont):
+def _add_blank_glyph_to_gid1(ttfont):
     from nanoemoji.reorder_glyphs import reorder_glyphs
     from nanoemoji.svg import _ensure_ttfont_fully_decompiled
-
-    second_glyph = ttfont.getGlyphOrder()[1]
+    from fontTools.ttLib.tables._g_l_y_f import Glyph
+    glyph_order = ttfont.getGlyphOrder()
     glyf_table = ttfont["glyf"]
-    # Will also work for composite glyphs since these have a -1 contour count
-    if glyf_table[second_glyph].numberOfContours > 0:
-        _ensure_ttfont_fully_decompiled(ttfont)
-        glyph_order = ttfont.getGlyphOrder()
-        empty_glyph = next(
-            (g for g in glyph_order if glyf_table[g].numberOfContours == 0),
-            None
-        )
-        if empty_glyph is None:
-            raise ValueError(
-                "Font contains no empty glyphs. Please include a space or .null glyph"
-            )
-        new_order = glyph_order
-        new_order.remove(empty_glyph)
-        new_order.insert(1, empty_glyph)
-        reorder_glyphs(ttfont, new_order)
+
+    gid1 = glyph_order[1]
+    if glyf_table[gid1].numberOfContours == 0:
+        return
+    _ensure_ttfont_fully_decompiled(ttfont)
+    hmtx = ttfont["hmtx"]
+    blank_glyph = Glyph()
+    blank_name = "blank"
```

Comment:
the name "blank" might already exist, you don't want to overwrite anything. Try to come up with a more unique name. ".null" if present is very likely to be empty, if no empty glyphs are found in the font, then naming the second glyph ".null" seems like a safer option than "blank"